### ISSUE TITLE
A4046 - CE-2-AI34: an input does not have associated label

### ISSUE THREAD

**Issue by RolandCampbell** [RolandCampbell/ForgetMeNot1#7]
*Common Element*: Offers bar\\ *Issue link*: [AI34|http://aqa.usablenet.com/training-and-demo/roa/1642259846e_a92df74362b1807cbeeadc9ac5b9ee97/A4046/T10232/step1/details/view/AI34]\\ *Description*: an input does not have associated label\\ *WCAG Success Criterion*: 1.3.1\\ *Element*: select\\ *Severity*: high\\ *Complexity*: easy
